eve/workers/pod.yml

```diff
@@ -15,7 +15,7 @@ spec:
         cpu: 500m
         memory: 3Gi
       limits:
-        cpu: "2"
+        cpu: 1500m
```

Comment:
Can you validate with @tcarmet that those limit changes are useful/meaningful?

Reply:
there's a 4 cpu limit on the pod Eve makes to run functional tests. Adding the redis image needed some rebalancing.

Reply:
@watsaqat is correct, those changes LGTM